package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@hospitalrun-org/frontend",
+  "version": "0.1.0",
+  "private": false,
+  "dependencies": {
+    "@types/jest": "24.0.18",
+    "@types/node": "12.7.4",
+    "@types/react": "16.9.2",
+    "@types/react-dom": "16.9.0",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
+    "react-scripts": "3.1.1",
+    "typescript": "3.6.2"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^8.1.0",
+    "@commitlint/core": "^8.1.0",
+    "commitlint-config-gitmoji": "^1.0.1",
+    "cross-env": "^5.2.1",
+    "cz-emoji": "^1.1.2",
+    "dateformat": "^3.0.3",
+    "husky": "^3.0.5"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "test:ci": "cross-env CI=true react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
```

Comment:
Is this eslintConfig compatible with TS? I can't see the language plugins.

Reply:
It's what the CRA outputs with the --typescript flag. 
- [TypeScript CRA](https://create-react-app.dev/docs/adding-typescript)